src/main/java/huan/task/DeadlineTask.java

```diff
@@ -0,0 +1,53 @@
+package huan.task;
+
+import java.util.Objects;
+
+public class DeadlineTask extends Task{
+    private String ddlTime;
```

Comment:
maybe can consider calling it deadlineTime, as ddlTime can be hard to understand what it means